### PR TITLE
added some more color/clarification for using consul discovery on gloo

### PR DESCRIPTION
**Pull request by christian-posta**

### CODE REVIEW

File: docs/content/gloo_routing/virtual_services/routes/route_destinations/consul_services/_index.md [hunk 1]
```diff
@@ -4,12 +4,78 @@ weight: 40
...
 
-A single Consul service usually maps to several service instances, which can have distinct sets of tags, listen on 
-different ports, and live in multiple data centers. To give a concrete example, here is a simplified response you might 
+When services registered into Consul require a bit more fine-grained grouping and routing, Gloo can do that too.
+
+## Configuring Gloo to Discover from Consul
+
+Gloo can automatically discover [Upstreams]({{% versioned_link_path fromRoot="/introduction/concepts#upstreams" %}}) from Consul. You can also explicitly create upstreams from Consul. 
+
+To enable automatic discovery of Consul services, update your `settings` CRD and add the `consul` section:
```
Comment (ned1313) [inline docs/content/gloo_routing/virtual_services/routes/route_destinations/consul_services/_index.md]: nit - CRD should be CR

**Comment by christian-posta** [comment]
@ned1313 PTAL. let's get this merged

**Pull request merged**